### PR TITLE
Added aflow and common resources.

### PR DESCRIPTION
**Pull request by devendra-bhat**

### CODE REVIEW

File: vendor/github.com/go_vthunder/vthunder/SlbAflow.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+type Aflow struct {
+	UUID AflowInstance `json:"aflow,omitempty"`
+}
+
+type SamplingEnable6 struct {
+	Counters1 string `json:"counters1,omitempty"`
+}
+type AflowInstance struct {
+	Counters1 []SamplingEnable6 `json:"sampling-enable,omitempty"`
+	UUID      string            `json:"uuid,omitempty"`
```
Comment (SwapnilKashidA10) [inline vendor/github.com/go_vthunder/vthunder/SlbAflow.go:20]: UUID should not be there

Comment (devendra-bhat) [reply vendor/github.com/go_vthunder/vthunder/SlbAflow.go:20]: We don't remove UUID from struct.

File: vthunder/resource_vthunder_slb_aflow_test.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package vthunder
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+var TEST_AFLOW_RESOURCE = `
+resource "vthunder_slb_template_aflow" "testname" {
```
Comment (SwapnilKashidA10) [inline vthunder/resource_vthunder_slb_aflow_test.go:10]: In recourse name template word is used

File: website/docs/r/vthunder_slb_aflow.html.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+
+```hcl
+provider "vthunder" {
+  address  = "129.213.82.65"
+  username = "admin"
+  password = "admin"
+}
+
+resource "vthunder_slb_template_aflow" "testname" {
```
Comment (SwapnilKashidA10) [inline website/docs/r/vthunder_slb_aflow.html.markdown:22]: Template word is used

**Pull request merged**